app.json

```diff
@@ -0,0 +1,15 @@
+{
+    "addons": [],
+    "buildpacks": [
+        {
+            "url": "heroku/python"
+        },
+        {
+            "url": "https://github.com/gerywahyunugraha/heroku-google-application-credentials-buildpack"
```

Comment:
I am curious if we want to fork this.  Actively linking it to someone else's repo seems a little bit risky.

Reply:
excellent suggestion.